index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>niicode</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;700&display=swap" rel="stylesheet"> 
+</head>
+<body>
+    <header>
+        <a href="/" class="logo-text font-style">My Logo</a>
+        <img class="menu-icon" src="/assets/images/Enabled.png" alt="blue menu icone">
```

Comment:
- [ ] [_Optional_] Kindly consider using a `nav` tag to encapsulate the hamburger since hamburger menus are components that allow you to navigate throughout the page.